_data/jobs.yml

```
@@ -1,29 +1,29 @@
-- title: Full Stack Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/384247
-  description: Be part of a cross-functional team and use your Java and/or TypeScript skills to further develop our new ecommerce platform.
-  location: Hamburg
+- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
```

```suggestion
- title: Software Developer (m/f/d) - 3rd Party Integration, REST API
```
It is done like this for the internship post and I think it's nicer to put the gender information directly after the job title

---

@dkluever22 But the stuff after the - is part of the job title. It feels a bit odd and this is the actual job title HR put there. I would like to keep it consistent.

---

@AliceStatler Alright. I get that.